src/pages/index.js

```diff
@@ -31,7 +31,11 @@ class Index extends React.Component {
           This is a very long paragraph that contains a lot of information and lists it all out for the user to be able to see.
         </Paragraph>
         <Donut>
-          <Phone x={-300} y={110} width={300}/>
```

Comment:
same with this for css; I am not sure if you have to style it here or not. It would be cleaner if all styling was in the css file.